### PR TITLE
try: init at 0.1.0

### PR DESCRIPTION
**Pull request by pasqui23**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/admin/try/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    install -Dt $out/bin try
+    wrapProgram $out/bin/try --prefix PATH : ${lib.makeBinPath [ fuse-overlayfs util-linux ]}
+  '';
+  meta = with lib;{
+    homepage = "https://github/binpash/try";
+    description = "Lets you run a command and inspect its effects before changing your live system";
+    maintainers = with maintainers;[ pasqui23 ];
+    license = with licenses;[ mit ];
+    platforms = [ platforms.linux ];
+  };
```
Comment (ncfavier) [inline pkgs/tools/admin/try/default.nix:22]:
```suggestion
  meta = with lib; {
    homepage = "https://github.com/binpash/try";
    description = "Lets you run a command and inspect its effects before changing your live system";
    maintainers = with maintainers; [ pasqui23 ];
    license = with licenses; [ mit ];
    platforms = platforms.linux;
  };
```

File: pkgs/tools/admin/try/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  src = fetchFromGitHub {
+    owner = "binpash";
+    repo = pname;
+    rev = "b2df6b650cb2b589515631748761e9910a07c343";
+    hash = "sha256-oSc2j8nUfNR8KM8BCCGwtLVeIdFA00FNlffCMt9bfYo=";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+  installPhase = ''
+    install -Dt $out/bin try
+    wrapProgram $out/bin/try --prefix PATH : ${lib.makeBinPath [ fuse-overlayfs util-linux ]}
```
Comment (ncfavier) [inline pkgs/tools/admin/try/default.nix:14]:
```suggestion
    runHook preInstall

    install -Dt $out/bin try
    wrapProgram $out/bin/try --prefix PATH : ${lib.makeBinPath [ fuse-overlayfs util-linux ]}

    runHook postInstall
```

**Review by ncfavier — CHANGES_REQUESTED** [review]

File: pkgs/tools/admin/try/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{ stdenvNoCC, lib, bash, fetchFromGitHub, fuse-overlayfs, util-linux, makeWrapper }:
+stdenvNoCC.mkDerivation rec{
+  pname = "try";
+  version = "unstable-2023-06-25";
```
Comment (ncfavier) [inline pkgs/tools/admin/try/default.nix:4]: 0.1.0 has just been released

File: pkgs/tools/admin/try/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{ stdenvNoCC, lib, bash, fetchFromGitHub, fuse-overlayfs, util-linux, makeWrapper }:
```
Comment (ncfavier) [inline pkgs/tools/admin/try/default.nix:1]: `bash` is unused

File: pkgs/tools/admin/try/default.nix (hunk 3 above)
Comment (pasqui23) [reply pkgs/tools/admin/try/default.nix:4]: Upstream has no git tag like "0.1.0" nor "v0.1.0"

Comment (ncfavier) [reply pkgs/tools/admin/try/default.nix:4]: https://github.com/binpash/try/releases/tag/v0.1.0

**Review by ncfavier — APPROVED** [review]

**Pull request merged**